Update wallet docs for version 0.11 (#29)
vegaprotocol/documentation: docs/tools/vega-wallet/CLI-wallet/latest/migrate-0-10-0-to-0-11-0.md

## Create a wallet

**Up to version 0.11.0** 

Creating a wallet was done using the command:

```sh
vegawallet key generate --wallet WALLET_NAME
```

**Version 0.11.0 onwards**

Create a wallet using the command:

```sh
vegawallet create --wallet WALLET_NAME
```

For ease of use, the `create` command generates the first key.

**Deprecation** 

Creation of the wallet using the `key generate` command is deprecated and will be removed in version 0.12.0.

## Restore a wallet

**Version 0.11.0 onwards**

Using the `import` command generates the first key, just as the `create` command does.

**Deprecation** 

The `--mnemonic-file` flag has been deprecated in favour of the `--recovery-phrase-file` flag. The `--mnemonic-file` flag will be removed in version 0.12.0.

In the HTTP request for creating a wallet, the `mnemonic` property has been deprecated in favour of the `recoveryPhrase` property and will be removed in version 0.12.0.

In the HTTP reponse for importing a wallet, the `mnemonic` property has been deprecated in favour of the `recoveryPhrase` property and will be removed in version 0.12.0.

## Connect to a site using a proxy

**Up to version 0.11.0** 

> In 0.10.0, starting the Vega Console through your wallet was acheived using the `--console-proxy` flag. 

Reviewer: ```suggestion
In 0.10.0, starting the Vega Console through your wallet was achieved using the `--console-proxy` flag. 
```

**Version 0.11.0 onwards**

Starting the Vega Console through your wallet is done using the `--with-console` flag. 

It's now possible to proxy the Vega Token dApp from the command line with the `--with-token-dapp` flag.

Both sites can be started in a single command:

```sh
vegawallet service run --network NETWORK --with-console --with-token-dapp
```

**Deprecation** 

`--console-proxy` is deprecated in favour of `--with-console` and doesn't have a short version anymore. The old command will continue to work until version 0.12.0 is released.

## Send a command or transaction

**Up to version 0.11.0** 

The command used to send a command or a raw transaction was:

```sh
vegawallet send command ...
vegawallet send tx ...
```

**Version 0.11.0 onwards** 

> It now matches the rest of the command line semantic:

Reviewer: ```suggestion
It now matches the rest of the command line semantics:
```

```sh
vegawallet command send ...
vegawallet tx send ...
```

**Deprecation** 

> The `send command` and `send transaction` semantic will not be available from version 0.12.0, and only `command send` and `tx send` will be available.

Reviewer: ```suggestion
The `send command` and `send transaction` commands will not be available from version 0.12.0, and only `command send` and `tx send` will be available.
```

## More guidance in help

> The `--help` flags across commands offer more information about each command and sub-command.  You'll see an explanation of each command and examples of how to use them. 

Reviewer: ```suggestion
The `--help` flag offers more information about each command and sub-command.  You'll see an explanation of each command and examples of how to use them. 
```
